src/main/java/pages/CartPage.java

```diff
@@ -0,0 +1,56 @@
+package pages;
+
+import Constants.Constants;
+import org.openqa.selenium.By;
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+
+public class CartPage extends BasePage implements Constants {
+    public CartPage(WebDriver driver) {
+        super(driver);
+    }
+
+    @Override
+    public CartPage waitForPageOpened() {
+        try {
+            wait.until(ExpectedConditions.visibilityOfElementLocated(CHECKOUT_BUTTON));
+        } catch (TimeoutException exception) {
+            Assert.fail(String.format("Cart Page page it's not loaded! Locator: '%s' was not found!", CHECKOUT_BUTTON));
+        }
+        return this;
+    }
+
+    public CartPage openPage() {
+        openPage(URL_SOUSE_DEMO+URL_CART_PAGE);
```

Comment:
use Ctrl+Shift+L to format your code properly.
URL_SOUSE_DEMO(space)+(space)URL_CART_PAGE

Reply:
fix